app/src/main/java/org/schabi/newpipe/player/helper/MediaSessionManager.java

```diff
@@ -40,13 +49,45 @@ public KeyEvent handleMediaButtonIntent(final Intent intent) {
         return MediaButtonReceiver.handleIntent(mediaSession, intent);
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public void setLockScreenArt(NotificationCompat.Builder builder, @Nullable Bitmap thumbnailBitmap) {
+        if (thumbnailBitmap == null || !mediaSession.isActive()) {
+            return;
+        }
+
+        mediaSession.setMetadata(
+                new MediaMetadataCompat.Builder()
+                        .putBitmap(MediaMetadata.METADATA_KEY_ALBUM_ART, thumbnailBitmap)
+                        .build()
+        );
+
+        MediaStyle mediaStyle = new MediaStyle()
+                .setMediaSession(mediaSession.getSessionToken());
+
+        builder.setStyle(mediaStyle);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public void clearLockScreenArt(NotificationCompat.Builder builder) {
+        mediaSession.setMetadata(
+                new MediaMetadataCompat.Builder()
+                        .putBitmap(MediaMetadata.METADATA_KEY_ALBUM_ART, null)
+                        .build()
+        );
+
+        MediaStyle mediaStyle = new MediaStyle()
+                .setMediaSession(mediaSession.getSessionToken());
+
+        builder.setStyle(mediaStyle);
+    }
+
     /**
-     * Should be called on player destruction to prevent leakage.
-     * */
+     * Should be called on player destruction to prevent leakage.BitmapUtils
```

Comment:
Why is there `BitmapUtils` at the end of the comment?
```suggestion
     * Should be called on player destruction to prevent leakage.
```

Reply:
Fixed.

Reply:
I already fixed this in https://github.com/TeamNewPipe/NewPipe/pull/2690/commits/1aa763e86cb24a8dbda55f7a7576c30a7618a440. Something went wrong in your meged commit: https://github.com/TeamNewPipe/NewPipe/pull/2690/commits/835476870b7b4a04af3f79c6c5ecb0c9b70e588f. This branch was on top of the dev branch already